tests/Fixture/DatabaseLogsFixture.php

```diff
@@ -26,13 +26,13 @@ class DatabaseLogsFixture extends TestFixture {
 		'type' => ['type' => 'string', 'null' => false, 'default' => null, 'length' => 50],
 		'message' => ['type' => 'text', 'null' => false, 'default' => null],
 		'context' => ['type' => 'text', 'null' => true, 'default' => null],
-		'created' => ['type' => 'datetime', 'null' => true, 'default' => null],
+		'created' => ['type' => 'timestamp', 'null' => true, 'default' => null],
```

Comment:
This should stay datetime afaik.

Reply:
https://github.com/dereuromark/CakePHP-DatabaseLog/blob/master/config/Migrations/20161005230232_InitDatabaseLogs.php#L36
is maybe the wrong one?

Reply:
Given the fact that the current code uses timestamp I would merge this now. We should probably fix the migration in general in a feature minor release.

Reply:
Thank you @dereuromark